io.github.torbuntu.Leikr.json

```diff
@@ -0,0 +1,42 @@
+{
+    "id": "io.github.torbuntu.Leikr",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "command": "/app/Leikr/Leikr",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.openjdk"],
+    "finish-args": [
+        "--env=PATH=/app/jre/bin:/usr/bin",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-config",
```

Comment:
```suggestion
```
There's no such thing, you have access to your .var/app/$app-id which is a per app equivalent to .local/share.

Reply:
My project requires access to the xdg-config directory in order to store user modifiable content in the `~/.config/Leikr` directory. Without this the application cannot access that location.

Reply:
Any reason for hardcoding that path?

Reply:
Consistency. The project itself is a Java game and uses `System.getProperty("user.home")+"/.config.Leikr/"` in order to always have the files maintained but accessible to users. 
The project also sets a java [security policy ](https://github.com/Torbuntu/Leikr/blob/master/assets/Sys/mysecurity.policy) in order to restrict access to that directory.

Reply:
Well, the restriction is also handled by Flatpak's sandbox. As apps don't have access to other apps's configurations. For that to work properly, you should be using $XDG_CONFIG_HOME instead of hardcoding the config dir to $HOME/.config. You can keep that as a fallaback though if you want.

Reply:
After lots of discussion with some much smarter folks than myself, it became apparent that this was the wrong place to store the data my application uses. Instead, like Arduino IDE, ApacheNetBeans, and others, it should be a folder on $HOME for accessing project files directly. So I updated to accommodate that change.